src/app/components/menu/menu.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { TokenService } from 'src/app/services/token.service';
+
+@Component({
+  selector: 'app-menu',
+  templateUrl: './menu.component.html',
+  styleUrls: ['./menu.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7132249)
```suggestion
  styleUrls: ["./menu.component.css"]
```